tools/protovalidate-conformance/internal/cases/cases_message.go

```diff
@@ -155,5 +155,36 @@ func messageSuite() suites.Suite {
 				},
 			),
 		},
+		"oneof/single-field/valid": {
+			// BoolField is not part of the oneof, so it can be set on this message
+			Message:  &cases.MessageOneofSingleField{StrField: "test", BoolField: true},
+			Expected: results.Success(true),
+		},
+		"oneof/two-fields/valid": {
+			Message:  &cases.MessageOneofMultipleFields{StrField: "test"},
+			Expected: results.Success(true),
+		},
+		"oneof/two-fields/invalid": {
+			Message: &cases.MessageOneofMultipleFields{StrField: "test", BoolField: true},
+			Expected: results.Violations(
+				&validate.Violation{
+					RuleId:  proto.String("message.oneof"),
+					Message: proto.String("only one of ['str_field', 'bool_field'] can be set"),
```

Comment:
I'd prefer that we format as `only one of str_field, bool_field can be set` because these field names are identifiers.